managed/CounterStrikeSharp.API/Core/GlobalContext.cs

```diff
@@ -124,27 +130,149 @@ public void LoadAll()
 
             foreach (var path in filePaths)
             {
-                var plugin = new PluginContext(path);
-
+                if (path.Contains("disabled")) continue;
```

Comment:
I don't think this is super required, as we only match against `{PluginName}/{PluginName}.dll` to load plugins, so the user could move the plugin into a disabled subfolder and it would immediately not match I think.

Reply:
Just tried this - this line isn't required.